### PR TITLE
First move of asides to separate text blocks

### PR DESCRIPTION
**Pull request by rylev**
As discussed in the RFC https://github.com/intermezzOS/rfcs/pull/3, we decided to move all asides to separate text blocks. Using the basic block quote style `> TEXT GOES HERE`, I got the following results. Not sure if it contrasts enough with the rest of the text to be good enough. Thoughts?

screen shot 2016-01-04 at 22 00 29

### CODE REVIEW

**Comment by azerupi** [comment]
I probably messed something up, this is not how they were supposed to look. I will check what went wrong and re-upload a new version of mdBook in the following days. Sorry for that :/

In the mean time, this can be merged as the style will just be updated at the next release of mdBook

File: src/multiboot-headers.md [hunk 1]
```diff
@@ -68,16 +67,18 @@ its findings.
...
-booting process behind some sort of splash screen. It's usually possible to see the
-BIOS' logs by pressing some collection of keys when your computer is starting up.
-
-The BIOS also has a menu where you can see information about the computer
-like CPU and memory specs and all the hardware the BIOS detected like hard drives
-and CD and DVD drives. Typically this menu is accessed by pressing some other
-weird collection of keyboard keys while the computer is attempting to boot.
-
-Next, the BIOS automatically finds a ‘bootable drive’ by looking in certain
+> **By the way...*
```
Comment (steveklabnik) [inline src/multiboot-headers.md]: missing a trailing \* here

**Comment by steveklabnik** [comment]
>  Not sure if it contrasts enough with the rest of the text to be good enough.

Yeah, so this is an improvement, but I'm not sure that the general quote mechanism is good enough for asides or not. I'm more interested in the _semantics_ here than the actual visual contrast, which is mostly about tweaking CSS.

Regardless, this is a good intermediate step, so if you add in that trailing asterisk, i'll merge :)

**Comment by rylev** [comment]
@steveklabnik Good to go. Let me know if I can help tweaking anything or experimenting with different styles or trying a new mechanism besides block quoting.

**Comment by steveklabnik** [comment]
Yeah, I'm not sure. What about some kind of handlebars helper that emits a `<section>` with an aside class? Hmm

**Comment by sarahhodne** [comment]
Not to bikeshed this, but HTML also has the `<aside>` tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside.

**Comment by steveklabnik** [comment]
Oh right!

**Comment by azerupi** [comment]
I thought block quotes would be easier, and they are used in the current Rust book for aside-ish notes. If you prefer I can style the `aside` tag in mdBook and you can then just use normal html in between the markdown :)

``` html
<aside>
    Some aside
</aside>
```

Or you can do it here, if you create a css file in the root directory containing only the tweaks you need, and include it in the custom template after the inclusion of `book.css` it should work (I haven't tested though). You could also override `book.css` directly in the `theme` directory but the stylesheet is updated a lot more often than the template (upstream), so you will still get the style updates if you override only the template.

> handlebars helper that emits a `<section>` with an aside class?

Unfortunately, handlebars helpers don't work on the markdown input (yet?). The playpen feature that looks like a handlebars helper is actually manually parsed. I may change that when I do the big refactoring.

**Pull request merged**